phpstan.dist.neon

```diff
@@ -17,6 +17,11 @@ parameters:
     reportPossiblyNonexistentGeneralArrayOffset: true # enabled, but huge exceptions count; example fix: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8089
     reportPossiblyNonexistentConstantArrayOffset: true
     treatPhpDocTypesAsCertain: false
+    typeAliases:
+        _PhpTokenArray: 'array{0: int, 1: non-empty-string}'
```

Comment:
the `int` should be improved as well as the token IDs are known as well - `T_*`

Reply:
step by step (or file another PR pls)